ubi_manifest/worker/tasks/depsolver/utils.py

```diff
@@ -6,13 +6,18 @@
 from typing import Dict, List, Tuple
 
 from pubtools.pulplib import Client, Criteria, Matcher, RpmDependency
-from rpm import labelCompare as label_compare  # pylint: disable=no-name-in-module
 from ubiconfig import UbiConfig
 
 from ubi_manifest.worker.tasks.depsolver.models import PackageToExclude
 
 _LOG = getLogger(__name__)
 
+try:
+    from rpm import labelCompare as label_compare  # pylint: disable=no-name-in-module
+except ImportError as ex:  # pragma: no cover
+    _LOG.error("rpm.labelCompare import working")
```

Comment:
This message could be a bit better.
Can you change it to something like
`Cannot import rpm module, please install rpm python bindings`